public/scripts/controllers/camp_manage.js

```diff
@@ -44,6 +44,18 @@ app.controller("manageCampsController", function ($scope, $http, $filter) {
         }
     }
 
+    // update the camp pre sale quota
+    $scope.updatePreSaleQuota = (camp_id,quota) => {
+        confirm('Confirm new quota to: ' + quota);
+        $.post('/camps/' + camp_id + '/updatePreSaleQuota', { quota: quota })
+           .success(function(response) {
+            window.location.reload();
+        })
+        .error(function() {
+            alert("Quota must be in a positive number format");
```

Comment:
What is the alert for?
If it's for debugging, you can use console.log, since it's wrapped by winston, which can do some nice stuff, like writing logs to multiple destinations and control trace levels from config.

If it's for the user, I think you should use sweetAlert.